### DISCUSSION TITLE
Flexoki Dark Theme

### DISCUSSION THREAD

**Discussion by nema69** [TriliumNext/Trilium#4324] [Show and tell]
Hey everyone, I decieded to use the flexoki dark color palette to make a theme for Trilium. 
https://github.com/nema69/trilium_flexoki
Highlights include a margin between the leftmost tab and the launcher bar and a smaller global menu button (trilium logo button). 
If anyone has ideas for improvements please reach out, this is not something I've done before.

**Comment by zadam** [comment]
Nice!
